### PR TITLE
New: Card Component

### PR DESCRIPTION
**Pull request by omgaz**
- Card Container Component
  + Similar to bordered well but with padding, box-shadow, content/container based, and contextual
- Card Content Component
  + Defaults with 16px padding
  + Content can, optionally, stretch (width) and fill (width and height) component
- Cards can be inline or stacked

screen shot 2016-05-25 at 15 40 47

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM

**Comment by etaoins** [comment]
JavaScript and tests look good to me :100:

File: src/components/Main.js [hunk 1]
```diff
@@ -114,6 +115,33 @@ class AppComponent extends React.Component {
...
         <Breadcrumb nodes={this.state.breadcrumbNodes} onClick={this.breadcrumbOnClick} />
 
+        <h1>Cards</h1>
+        <Card.Container>
+          <Card.Content><small>I am Cardy McCardface.</small></Card.Content>
+          <Card.Content><em>I am Cardy McCardface.</em></Card.Content>
+          <Card.Content><strong>I am Cardy McCardface.</strong></Card.Content>
+        </Card.Container>
+        <Card.Container inline>
+          <Card.Content fill><PageTitle title="Card" /></Card.Content>
```
Comment (OndrejCholeva) [inline src/components/Main.js]: Can it be `Card with long text that wraps` to show that fill works?

Comment (omgaz) [inline src/components/Main.js]: Wrapping would be an implementation detail based on a set width of a card. Cards in Alexandria do not dictate width and so would flow to be as big as their container (in this case 940px). The integration point will decide the width of the card and how the content inside will behave e.g. wrap/no-wrap, ellipsis etc…

**Comment by davepoon** [comment]
Have you thought of adding a `CardTitle` and `CardAction`(Buttons or Links) components in the `Card `component?
Or you prefer to simply adding html and button inside `CardContent` instead of creating `CardTitle` and `CardAction` components?

**Comment by omgaz** [comment]
@davepoon Yup, my thoughts exactly. Similar to my content to Ondrej, I'm not keen on dictating (too much) what goes on inside the card. If we run into 'necessities' further down the line we'll tackle them at that point in time.

As the screenshot shows, they're pretty versatile. The Adslot and Symphony products will be able to manipulate these to have `<MarketplaceCard>` etc… with all the various details.

**Comment by davepoon** [comment]
@omgaz Cool~
Javascript and SASS LGTM :thumbsup:

**Comment by vinteo** [comment]
Merging...

**Pull request merged**